pkg/minikube/config/types.go

```diff
@@ -66,6 +66,10 @@ type ClusterConfig struct {
 	HostDNSResolver         bool   // Only used by virtualbox
 	HostOnlyNicType         string // Only used by virtualbox
 	NatNicType              string // Only used by virtualbox
+	IPAddress               string // Only used by ssh
```

Comment:
I think this should be called SSHIPAddress or SSHRemoteIP
 for clarity, so it doesn't get mixed with node IP or master IP...